autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/AutofillCapabilityChecker.kt

```diff
@@ -50,4 +50,9 @@ interface AutofillCapabilityChecker {
      * Whether autofill is configured to be enabled. This is a configuration value, not a user preference.
      */
     suspend fun isAutofillEnabledByConfiguration(url: String): Boolean
+
+    /**
+     * Whether canCategorizeUnknownUsername feature is configured to be enabled.
+     */
+    suspend fun canCategorizeUnknownUsername(): Boolean
```

Comment:
discussed this offline. 
- i don't think this needs to be in the public API. 
- even though it's consistent with the other functions, don't want to add to the public API (have plans to reduce it already)
- alternative: `RealAutofillRuntimeConfigProvider` can check `AutofillFeature` directly, and that keeps everything inside of `autofill-impl`